### PR TITLE
APIGOV-24134 - Category in multi-marketplaces

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Added the category with multiple marketplaces

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | e9d5f5a |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-269.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by lbadenhop — APPROVED** [review]

File: content/en/docs/manage_product_foundry/foundry_product_tags_attributes_categories.md [hunk 1]
```diff
@@ -30,15 +30,31 @@ Categories allow for a broader grouping of products into different topics or the
 
 The category types are:
 
-* **Global**: These are either legacy categories that were created using the Unified Catalog category management system, or are categories created while creating products. They can be used with either Asset or Products.
-* **Product specific**:  These categories can only be assigned to a product.
+* **Global**: Either legacy categories that were created using the Unified Catalog category management system, or are categories created while creating products. They can be used with either Asset or Products.
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/foundry_product_tags_attributes_categories.md:33]: or categories added while creating products

File: content/en/docs/manage_product_foundry/foundry_product_tags_attributes_categories.md [hunk 2]
```diff
@@ -30,15 +30,31 @@ Categories allow for a broader grouping of products into different topics or the
...
+* **Product specific**: Categories that can only be assigned to a product.
 
 The product specific categories have two types of visibility:
 
-* Private: This category is used for filtering the Product Foundry catalog only.
-* Public: This category is used to filter the Product Foundry catalog, as well as the Marketplace product.
+* Private: For filtering the Product Foundry catalog only.
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/foundry_product_tags_attributes_categories.md:38]: * Private: Visible in the Product Foundry only.
* Public: Visible both in the Product Foundry and the Marketplace.

File: content/en/docs/manage_product_foundry/foundry_product_tags_attributes_categories.md [hunk 3]
```diff
@@ -70,10 +85,20 @@ Only users that are assigned the **Central Admin** role can edit product categ
 Only product categories, not global categories, can be edited from the WebUI.
 
 1. Log into the platform.
-2. Navigate to **Catalog > Product Foundry > Categories**. This page displays all global categories assigned only to products and product categories.
-3. Open the ellipsis menu on a category and select **Edit**. The category wizard opens in Edit mode to enable changes to all information except the Category logical name.
+2. Navigate to *Catalog > Product Foundry > Categories*. This page displays all global categories assigned only to products and product categories.
+3. Open the ellipsis menu on a category and select **Edit**. The category wizard opens in edit mode to enable changes to all information except the Category logical name.
 4. Save the changes.
 
+### Converting a product category into a Featured category
```
Comment (georgiana-caras) [inline content/en/docs/manage_product_foundry/foundry_product_tags_attributes_categories.md:92]: Setting a product category as featured in a Marketplace

**Pull request merged**